context/app/static/js/components/Detail/Attribution/Attribution.spec.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { render } from '../../../test-utils';
```

Comment:
Set up a webpack alias for this, to avoid the long relative paths?

Reply:
Tests are being run outside of webpack, but I'll work to fix it.